python/servo/command_base.py

```diff
@@ -313,6 +327,9 @@ def build_env(self, gonk=False, hosts_file_path=None):
             if subprocess.call(['which', 'ld.gold'], stdout=PIPE, stderr=PIPE) == 0:
                 env['RUSTC'] = path.join(self.context.topdir, 'etc', 'rustc-with-gold')
 
+        # This sometimes ends up as unicode
+        env["PATH"] = env["PATH"].encode('ascii', 'ignore')
```

Comment:
I don't really know enough about string encoding in Python to say what the right thing to do here is.